wordcount/src/wordcount/Main.java

```diff
@@ -0,0 +1,59 @@
+
+		// theres 2 ways i can think of to do this. this is the first way
+		// the space in-between .split(" ") is needed to properly split out the un wanted characters.
+		System.out.println("looks like there are " + humanRights.split(" ").length + " words in the united nations declaration of human rights");
+
+		// way 2 is what you were really looking for though
+		// first we have to replace illegal or unwanted characters and split the words into an array
+		System.out.println("*** Lets do this the right way ***");
+		humanRights.replaceAll("[[\\.\\?\\!\\,\\;\\:\\{\\}\\(\\)\\']]", "");
+		String[] removedIllegals = humanRights.split(" +");
```

Comment:
Line 19 - Make sure to assign the result of the `.replaceAll` method to the `humanRights` variable - it returns a String value but does not mutate the original String, which is why you'll still have to assign it like so:
```
humanRights = humanRights.replaceAll("[[\\.\\?\\!\\,\\;\\:\\{\\}\\(\\)\\']]", "");
```

Take note that **not assigning the return value will cause the punctuation marks to be retained, potentially altering the results given when running your code**.